src/main/java/roomescape/reservation/service/ReservationService.java

```diff
@@ -0,0 +1,69 @@
+package roomescape.reservation.service;
+
+import com.sun.jdi.request.DuplicateRequestException;
+import org.springframework.stereotype.Service;
+import roomescape.error.exception.PastDateTimeException;
+import roomescape.error.exception.ReservationTimeNotExistsException;
+import roomescape.error.exception.ThemeNotExistsException;
+import roomescape.reservation.Reservation;
+import roomescape.reservation.dto.ReservationRequest;
+import roomescape.reservation.dto.ReservationResponse;
+import roomescape.reservationTime.ReservationTime;
+import roomescape.reservationTime.service.ReservationTimeRepository;
+import roomescape.theme.Theme;
+import roomescape.theme.service.ThemeRepository;
+
+import java.util.List;
+import java.util.Optional;
+import java.util.stream.Collectors;
+
+@Service
+public class ReservationService {
+
+    private final ReservationRepository reservationRepository;
+    private final ReservationTimeRepository reservationTimeRepository;
+    private final ThemeRepository themeRepository;
+
+    public ReservationService(ReservationRepository reservationRepository,
+        ReservationTimeRepository reservationTimeRepository, ThemeRepository themeRepository) {
```

Comment:
```suggestion
    public ReservationService(ReservationRepository reservationRepository, 
                              ReservationTimeRepository reservationTimeRepository,
                              ThemeRepository themeRepository) {
```

줄이 길어졌을 때 파라미터는 이렇게 관리해봐도 괜찮을 것 같아요.